app/oracle-server/oracle-server.sbt

```diff
@@ -0,0 +1,27 @@
+name := "bitcoin-s-oracle-server"
+
+// Ensure actor system is shut down
+// when server is quit
+Compile / fork := true
```

Comment:
nit: I don't think this stuff is needed, and i definitely don't think we need to fork for all compile tasks. You can read more about this here: https://www.scala-sbt.org/1.x/docs/Forking.html#Enable+forking

Reply:
I just copied from the app server's